Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.5-alpine
+FROM ruby:2.6.6-alpine
```

Comment:
Do we use this dockerfile for anything? 

One of the things that will be lost by the switch away from Jenkins is testing the dockerfile builds and pushing it dockerhub with each successful build. I assume we don't get value from either of those steps so fine to drop them. But since after that nothing will check the dockerfile builds, I wonder if there's point in keeping it?

Reply:
No, we don't. This repo was once a fork of `binaryberry/seal`, and that had a Dockerfile in it. So maybe it can be removed.